docs/reference/reference.md

```diff
@@ -10,30 +10,31 @@ The Dapr CLI allows you to setup Dapr on your local dev machine or on a Kubernet
   \__,_/\__,_/ .___/_/     
               /_/            
                                                                            
-======================================================
-A serverless runtime for hyperscale, distributed systems
+===============================
+Distributed Application Runtime
 
 Usage:
   dapr [command]
 
 Available Commands:
-  components     List all Dapr components
-  configurations List all Dapr configurations
+  completion     Generates shell completion scripts
+  components     Lists all Dapr components
```

Comment:
List

Reply:
Same.

Reply:
I've actually deleted these files since they are now in the docs website.